testing/MLDBFB-506-stats-tbl-sql-expr.py

```diff
@@ -0,0 +1,104 @@
+mldb = mldb_wrapper.wrap(mldb) # noqa
+
+
+class BowSqlExprTest(MldbUnitTest):
+
+    def check_true(self, q, truth):
```

Comment:
we have a dedicated assert equal query result function... check https://github.com/mldbai/mldb/blob/master/testing/MLDBFB-336-sample_test.py